tests/test_backup.py

```diff
@@ -366,6 +366,209 @@ def test_delete_backup(self, mock_available_backups, tmpdir, caplog):
         assert deleted is True
         assert child_backup.backup_id not in parent_backup.children_backup_ids
 
+        # Test 8: Update next rsync backup information
+        given_backup = build_test_backup_info(
+            backup_id="parent_backup_id",
+            server=backup_manager.server,
+        )
+        build_backup_directories(given_backup)
+        next_backup = build_test_backup_info(
+            backup_id="next_backup_id",
```

Comment:
```suggestion
        given_backup = build_test_backup_info(
            backup_id="rsync_backup_id",
            server=backup_manager.server,
        )
        build_backup_directories(given_backup)
        next_backup = build_test_backup_info(
            backup_id="next_rsync_backup_id",
```

I think this naming will provide a clearer idea of what we are expecting here.